docs/sql-migration-guide.md

```diff
@@ -24,6 +24,8 @@ license: |
 
 ## Upgrading from Spark SQL 3.1 to 3.2
 
+  - Since Spark 3.2, all the supported JDBC dialects use StringType for ROWID. In Spark 3.1 or earlier, Oracle dialect uses StringType and the other dialects use LongType.
```

Comment:
`dialects` is an internal word? If so, how about saying "Since Spark 3.2, \`java.sql.ROWID\` is mapped to \`StringType\` when reading data from other databases via JDBC"?

Reply:
At least, I use `dialects` here as a general word, not represent specific implementations like `PostgresDialect`.
`dialect` and `dialects` have used from before in the migration guide.